tests/test_vector2_normalize.py

```diff
@@ -0,0 +1,16 @@
+from math import hypot
+from ppb_vector import Vector2
+import pytest
+
+test_data = [
+    (Vector2(1, 0), Vector2(1, 0)),
+    (Vector2(2, 2), Vector2(2, 2) * (1 / hypot(2, 2))),
+    (Vector2(31, -15), Vector2(31, -15) * (1 / hypot(31, -15))),
```

Comment:
Could I get you to write the values expected explicitly? While I know this works, it does the same math that normalize does under the hood so is mostly proving that f(x) == f(x).